app/models/relationship.rb

```diff
@@ -380,6 +397,17 @@ def move_explainers_to_source
     end
   end
 
+  def apply_status_to_target
+    status = self.source.last_verification_status
+    target = self.target
+    s = target.last_status_obj
+    unless s.nil? || s.status == status
+      s.status = status
+      s.bypass_status_publish_check = true
+      s.save
+    end
```

Comment:
Sawy, I think this part can be refactored as its own instance method for the `Relationship` model, something like `sync_statuses`.

Reply:
Done